Content.Client/Robotics/UI/RoboticsConsoleWindow.xaml.cs

```diff
@@ -128,12 +128,12 @@ private void PopulateData()
         };
 
         var text = new FormattedMessage();
-        text.PushMarkup(Loc.GetString("robotics-console-model", ("name", model)));
-        text.AddMarkup(Loc.GetString("robotics-console-designation"));
+        text.AddMarkupOrThrow($"{Loc.GetString("robotics-console-model", ("name", model))}\n");
```

Comment:
Why the formatting changes here? They do not even do anything. And if so, they should be part of the localization string.
![grafik](https://github.com/user-attachments/assets/d2fe8818-1b44-4c02-88f9-fd4199d0d0c2)

Reply:
Nevermind, I missed that it was PushMarkup before